site/components/pages/Resources/Contact/index.tsx

```diff
@@ -0,0 +1,156 @@
+import { NextPage } from "next";
+import Link from "next/link";
+
+import {
+  PageWrap,
+  HeaderDesktop,
+  Footer,
+  NavItemDesktop,
+  HeaderMobile,
+  NavItemMobile,
+  ButtonPrimary,
+  Paragraph,
+  Section,
+} from "@klimadao/lib/components";
+import { Trans } from "@lingui/macro";
+
+import { urls } from "@klimadao/lib/constants";
+import styles from "./index.module.css";
+import { PageHead } from "components/PageHead";
+import { IS_PRODUCTION } from "lib/constants";
+import { FC, HTMLAttributes } from "react";
+import { Container } from "../Container";
+
+export type Props = HTMLHtmlElement & {};
+
+type H2Props = HTMLAttributes<HTMLHeadingElement> & {};
+
+const H2: FC<H2Props> = (props) => {
+  return (
+    <h2 {...props} className={styles.page_h2}>
```

Comment:
The parent won't be able to override classnames like this.
I believe you can do:

`className={cx(styles.page_h2, props.className)}`

Reply:
actually why do we need this component at all? Can't we target child `h2`s from the parent classes?

Reply:
Yeah, good point. I'll remove.